NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -167,8 +193,14 @@ class NeuroID private constructor(
     }
 
     @VisibleForTesting
-    fun setTestURL(endpoint: String){
-        nidJobServiceManager.endpoint = endpoint
+    fun setTestURL(newEndpoint: String){
+        endpoint = newEndpoint
```

Comment:
Is this assignment necessary?  Can `newEndpoint` be passed directly to the sender?